types/jsonwebtoken/index.d.ts

```diff
@@ -77,6 +78,19 @@ export interface DecodeOptions {
     complete?: boolean;
     json?: boolean;
 }
+
+export type SecretOrPublicKeyCallback = {
```

Comment:
This is not what I meant, this should be `export type SecretOrPublicKeyCallback  = (err: VerifyErrors, secretOrPublicKey: string | Buffer): void`

Reply:
My bad! I'll fix that right now.